docs/examples/Interacting.md

```diff
@@ -0,0 +1,236 @@
+  "featureDir": "features"
+},
+"sensinact.virtual.temperature": {
+  "name": "temp1",
+  "interval": 5000,
+  "latitude": 1.0,
+  "longitude": 2.0
+},
+...
+```
+
+We can query the value of the `sensor/temperature` resource as before at `http://localhost:8082/sensinact/providers/temp1/services/sensor/resources/temperature/GET`
+
+```js
+{
+  "type": "GET_RESPONSE",
+  "uri": "/temp1/sensor/temperature",
+  "statusCode": 200,
+  "response": {
+    "name": "temperature",
+    "timestamp": 1693394285012,
+    "type": "double",
+    "value": 29.60935577560438
+  }
+}
+```
+
+### Subscribing to updates
+
+The virtual temperature sensor periodically updates its temperature reading. This could be read by repeatedly polling the value, however this is inefficient, and risks missing changes if the update is more frequent than the polling interval.
+
+The SUBSCRIBE verb allows clients to register so that they can be notified when an update occurs, for the REST interface this is delivered as a Server Sent Event (SSE) stream, but other interfaces (such as Web Socket) may use other push mechanisms.
+
+Making a GET request to `http://localhost:8082/sensinact/providers/temp1/services/sensor/resources/temperature/SUBSCRIBE` will result in a series of `data` events being delivered:
+
+```js
+event: data
+data: {
+  "provider": "temp1",
+  "service": "sensor",
+  "resource": "temperature",
+  "timestamp": 1693402975313,
+  "oldValue": 18.758596432570382,
+  "newValue": 17.56708764568309
+}
+
+event: data
+data: {
+  "provider": "temp1",
+  "service": "sensor",
+  "resource": "temperature",
+  "timestamp": 1693402980313,
+  "oldValue": 17.56708764568309,
+  "newValue": 8.2679466364907
+}
+
+...
+```
+
+In the REST interface unsubscription occurs automatically when the client closes the SSE connection. For other northbound providers the initial subscription response will include a subscription identifier. This identifier can be passed back in an unsubscription request to terminate the delivery of notifications.
+
+### Updating values
+
+If a resource is `MODIFIABLE` that means that it can have its value updated by an end user using the `SET` verb. Unlike the other requests so far a `SET` operation in the REST API is a `PUT`, not a `GET`. The body of the PUT request is an array of parameters with a `name`, `type` and `value`. We can test this using `curl`
+
+```
+curl -d "[{\"name\": \"value\", \"type\": \"string\", \"value\": \"Lassie\"}]" -H "Content-Type: application/json" \
+http://localhost:8082/sensinact/providers/temp1/services/admin/resources/friendlyName/SET
+```
+
+A successful response will look like the following:
+
+```js
+{
+  "type": "SET_RESPONSE",
+  "uri": "/temp1/admin/friendlyName",
+  "statusCode": 200,
+  "response": {
+    "name": "friendlyName",
+    "timestamp": 1693407687936,
+    "type": "java.lang.String",
+    "value":"Lassie"
+  }
+}
+```
+
+After a `SET` subsequent `GET` requests will return the previously set value.
```

Comment:
Should we add a warning about pull/push-based resources which can have a different behavior?

Reply:
```suggestion
After a `SET` subsequent `GET` requests will return the previously set value.

> [!NOTE]
> If your resource is `FIXED` or `UPDATABLE` then it cannot be set by the northbound API. Also, the behaviour of `SET` requests may be slightly different if your resource is [pull-based](../southbound/custom/custom.md#GET-Methods) and the device returns a different value.
```

Reply:
Looks good to me